src/main/java/carpet/CarpetSettings.java

```diff
@@ -315,6 +315,9 @@ public String description() {
     )
     public static boolean hopperCounters = false;
 
+    @Rule( desc = "Allows Budding Amethyst blocks to be pushed by pistons", category = FEATURE )
+    public static boolean pushableAmethyst = false;
```

Comment:
Wouldn't `movableAmethyst` be more consistent, considering carpet's `movableBlockEntities` rule?

Reply:
Agreed, good thinking. I'll get on renaming